host-configs/lassen-blueos_3_ppc64le_ib_p9-clang@10.0.1.cmake

```diff
@@ -85,51 +85,51 @@ set(BLT_CMAKE_IMPLICIT_LINK_DIRECTORIES_EXCLUDE "/usr/tce/packages/gcc/gcc-4.9.3
+set(TPL_ROOT "/usr/WS2/smithdev/libs/serac/blueos_3_ppc64le_ib_p9/2023_06_23_16_51_25/clang-10.0.1" CACHE PATH "")
 
-set(AXOM_DIR "${TPL_ROOT}/axom-0.7.0.4-i6fqkv6wfzeunqj7iombno4ovrm7pihf" CACHE PATH "")
+set(AXOM_DIR "${TPL_ROOT}/axom-0.7.0.4-2tygx6zfdhpxqhd247szvz7vz5ivpt4o" CACHE PATH "")
 
-set(CAMP_DIR "${TPL_ROOT}/camp-2022.03.2-fxqxu6zxuto6ld4r626ndahdpukhjave" CACHE PATH "")
+set(CAMP_DIR "${TPL_ROOT}/camp-2022.03.2-preys5cwzspz6k44mynsst4bevpbx3ns" CACHE PATH "")
 
-set(CONDUIT_DIR "${TPL_ROOT}/conduit-0.8.4-3rwh3cymjdl7pbmrslfviinsi6dinlk3" CACHE PATH "")
+set(CONDUIT_DIR "${TPL_ROOT}/conduit-0.8.8-3d333zivvamspp4ztemuo4y232k25hle" CACHE PATH "")
 
-set(LUA_DIR "${TPL_ROOT}/lua-5.4.4-izgj5sjm3tx5otibr3fzzj5knx4ljcvw" CACHE PATH "")
+set(LUA_DIR "${TPL_ROOT}/lua-5.4.4-uj554fmtk6ayky6vmasbsenr4mpizatc" CACHE PATH "")
 
-set(MFEM_DIR "${TPL_ROOT}/mfem-4.5.3.1-gwcywbbebakx3xjjvz3cytvviohd5fji" CACHE PATH "")
+set(MFEM_DIR "${TPL_ROOT}/mfem-4.5.3.1-lr6n3uajnpqihrcftqaqj4fv6j5ela42" CACHE PATH "")
 
-set(HDF5_DIR "${TPL_ROOT}/hdf5-1.8.21-2anbh6idnvfadscqc4vftnjnitv7mm7f" CACHE PATH "")
+set(HDF5_DIR "${TPL_ROOT}/hdf5-1.8.21-rsj66pzlsswpvngt4ccpeql7677fhozk" CACHE PATH "")
 
-set(HYPRE_DIR "${TPL_ROOT}/hypre-2.26.0-i5vh2wqd6hiqbuki3x6l2owwrnsltmfj" CACHE PATH "")
+set(HYPRE_DIR "${TPL_ROOT}/hypre-2.26.0-meak4trkkki3ie26updimbceqvztmyec" CACHE PATH "")
 
-set(METIS_DIR "${TPL_ROOT}/metis-5.1.0-4qwkgow6wr7ibczuxiynqektxx525a53" CACHE PATH "")
+set(METIS_DIR "${TPL_ROOT}/metis-5.1.0-eqmtccvq5pnv536nyct3gfocjertrlyz" CACHE PATH "")
 
-set(PARMETIS_DIR "${TPL_ROOT}/parmetis-4.0.3-clhibcrftrsovvopqgossikfcom6pvtc" CACHE PATH "")
+set(PARMETIS_DIR "${TPL_ROOT}/parmetis-4.0.3-leavglocrsgja2o3jw2zkkdeoov5apfk" CACHE PATH "")
 
-set(NETCDF_DIR "${TPL_ROOT}/netcdf-c-4.7.4-3n3fmt2tzb4rbdp5wnypajwucybna53m" CACHE PATH "")
+set(NETCDF_DIR "${TPL_ROOT}/netcdf-c-4.7.4-g2xeohfuhubsfm6wtp3i4lntlg7qllgs" CACHE PATH "")
 
-set(SUPERLUDIST_DIR "${TPL_ROOT}/superlu-dist-6.1.1-rmetsgbcpj73j3jeipsesccashsblvxg" CACHE PATH "")
+set(SUPERLUDIST_DIR "${TPL_ROOT}/superlu-dist-6.1.1-rigrz4mlijd353hvplunxj4rbfsqwxwf" CACHE PATH "")
 
-set(ADIAK_DIR "${TPL_ROOT}/adiak-0.2.1-sasmwelsnyzgmshrqr2eha6ncsqj6sv5" CACHE PATH "")
+set(ADIAK_DIR "${TPL_ROOT}/adiak-0.2.2-4urg2fujp4yrmyvbougxkeoudjuqe7bp" CACHE PATH "")
 
-set(AMGX_DIR "${TPL_ROOT}/amgx-2.3.0.1-igdy23rgkevtkd5aibxklvlsyh6crpnn" CACHE PATH "")
+set(AMGX_DIR "${TPL_ROOT}/amgx-2.3.0.1-2ij5h7acreu5ktn37l5yb7kzyw6nuayz" CACHE PATH "")
 
-set(CALIPER_DIR "${TPL_ROOT}/caliper-2.7.0-v6whaccxqo6lmgx3mb6lsj47gjuwyink" CACHE PATH "")
+set(CALIPER_DIR "${TPL_ROOT}/caliper-2.9.0-zwlgzkslm5m57tcs7wbemsujzgyul74g" CACHE PATH "")
 
 # PETSC not built
 
-set(RAJA_DIR "${TPL_ROOT}/raja-2022.03.0-fczkcogd4ostziyulbiwzxa2frjwzguz" CACHE PATH "")
+set(RAJA_DIR "${TPL_ROOT}/raja-2022.03.0-vgn3f2l2nnwi5n74lzu2v6bqu5lcrjqo" CACHE PATH "")
```

Comment:
This did the same thing to you as it did to me. These are not the newest version of RAJA, CAMP or Umpire. I don't know why Spack is picking old versions but this is incredibly frustrating.

You need to lock down the versions with the `require` in the `spack.yaml` like this:

https://github.com/LLNL/axom/blob/dd5128c3e5d80e9974cfb0ee5ef2eef78e758d34/scripts/spack/configs/blueos_3_ppc64le_ib_p9/spack.yaml#L295-L310

Note, Axom's versions may be slightly ahead from Spack's repo because I moved to the radiuss-shared-spack-configs repo